ValheimPlus/Configurations/BaseConfig.cs

```diff
@@ -25,81 +29,115 @@ public string ServerSerializeSection()
             return r;
         }
 
-        public bool IsEnabled = false;
-        public virtual bool NeedsServerSync { get; set;} = false;
+        public bool IsEnabled { get; private set; } = false;
+        public virtual bool NeedsServerSync { get; set; } = false;
 
         public static IniData iniUpdated = null;
 
         public static T LoadIni(IniData data, string section)
         {
             var n = new T();
 
-            
+
             Debug.Log($"Loading config section {section}");
             if (data[section] == null || data[section]["enabled"] == null || !data[section].GetBool("enabled"))
             {
                 Debug.Log(" Section not enabled");
                 return n;
             }
-
-            n.LoadIniData(data[section]);
+            var keyData = data[section];
+            n.LoadIniData(keyData, section);
 
             return n;
         }
 
-        public void LoadIniData(KeyDataCollection data)
+        public void LoadIniData(KeyDataCollection data, string section)
         {
             IsEnabled = true;
+            var thisConfiguration = GetCurrentConfiguration(section);
+            if (thisConfiguration == null)
+            {
+                Debug.Log("Configuration not set.");
```

Comment:
Will `Debug.log` appear in user's `player.log`? 

If it does, then we should remove these to not pollute the log.

It it doesn't, then leaving them in sounds fine to me.